src/js/table.js

```diff
@@ -102,29 +102,31 @@ const cols = [
         type: 'detail',
         property_name: 'vat',
         is_numeric: true,
-        help: 'Caution: when stuff is not supplied by Amazon, then tax is often not listed.',
+        help: 'Caution: tax is often not listed when stuff is not supplied by Amazon, is cancelled, or is pre-order.',
         sites: new RegExp('amazon(?!.com)')
     },
     {
-        field_name: 'TAX',
+        field_name: 'tax',
         type: 'detail',
         property_name: 'us_tax',
         is_numeric: true,
-        help: 'Caution: when stuff is not supplied by Amazon, then tax is often not listed.',
+        help: 'Caution: tax is often not listed when stuff is not supplied by Amazon, is cancelled, or is pre-order.',
         sites: new RegExp('\\.com$')
     },
     {
         field_name: 'GST',
         type: 'detail',
         property_name: 'gst',
         is_numeric: true,
+        help: 'Caution: tax is often not listed when stuff is not supplied by Amazon, is cancelled, or is pre-order.',
```

Comment:
not mandatory - I'm going to merge this if I don't find a show stopper...
we could do: `const tax_help = "blah"` and then`help: tax` everywhere it's needed, to avoid the cost of keeping them all in sync.